opaque_keys/edx/keys.py

```diff
@@ -4,16 +4,11 @@
 from __future__ import annotations
 import json
 from abc import abstractmethod
-from typing import TYPE_CHECKING, Self
+from typing import Self
 import warnings
 
-from typing_extensions import deprecated  # For python 3.13+ can use 'from warnings import deprecated'
-
 from opaque_keys import OpaqueKey
 
-if TYPE_CHECKING:
-    from opaque_keys.edx.locator import LibraryLocatorV2
```

Comment:
This refactor allows us to get rid of this messy import of a locator (subclass) in the keys (parent class) file.